apps/site/lib/site_web/templates/customer_support/_form.html.eex

```diff
@@ -64,9 +64,10 @@
         </div>
       </div>
       <div class="plan-time-select">
-        <%= label f, "Vehicle", [id: "vehicleLabel", for: "support_vehicle", class: "form-control-label"] %>
-        <div>
-          <%= text_input(f, :vehicle, placeholder: placeholder_text("vehicle"), class: "plan-date-input") %>
+        <%= support_error_tag @errors, "vehicle" %>
+        <div class="form-group <%= class_for_error("vehicle", @errors, "has-danger", "has-success") %>">
+          <%= label f, "Vehicle", [id: "vehicleLabel", for: "vehicle", class: "form-control-label"] %>
+          <%= text_input(f, :vehicle, id: "vehicle", placeholder: placeholder_text("vehicle"), class: "form-control plan-date-input", value: assigns[:vehicle] ) %>
```

Comment:
I realize this was there before, but is `plan-date-input` the right class? Since it's not a date input.

Reply:
There's this and a few other classes borrowing styling from various inputs in the Trip Planner.. I'll clean these up in a followup commit